openshift/commonPipeline.groovy

```diff
@@ -123,7 +123,7 @@ def runStageDeploy(String stageEnv, String projectEnv, String hostEnv, String pa
       openshift.selector('cm', "getok-server-config").exists() &&
       openshift.selector('secret', "getok-keycloak-secret").exists() &&
       openshift.selector('secret', "getok-sc-ches-secret").exists() &&
-      openshift.selector('secret', "getok-sc-keycloakint-secret").exists() &&
+      openshift.selector('secret', "getok-sc-keycloakdev-secret").exists() &&
```

Comment:
I think we should have this be `getok-sc-keycloak-dev-secret` instead (and updated for everything else referencing it in order to be consistent with all the other secret nomenclatures.

Reply:
Ok. I can wrap this as a part of the upcoming db seed changes as that will do some pipeline modifications anyways.